src/view/invoice.js

```diff
@@ -38,13 +46,22 @@ const InvoiceView = ({ store, nav, invoice }) => (
     <MainContent>
       <Card>
         <BalanceLabel style={styles.balance}>
+          <BalanceLabelNumeral
+            style={[
+              styles.fiatUnit,
+              { fontSize: store.settings.displayFiat ? font.sizeXXXL : 0 },
+            ]}
+          >
+            $
+          </BalanceLabelNumeral>
```

Comment:
What happens for Euro or GBP? Probably best to rename the current fiat `display` names to `displayLong` in config.js like fot the btc units and add a short `display` for the currency symbols being displayed.